examples/CarHandling/CarHandlingDemo.cpp

```diff
@@ -0,0 +1,423 @@
+/*
+Bullet Continuous Collision Detection and Physics Library
+Copyright (c) 2003-2006 Erwin Coumans  http://continuousphysics.com/Bullet/
+
+This software is provided 'as-is', without any express or implied warranty.
+In no event will the authors be held liable for any damages arising from the use of this software.
+Permission is granted to anyone to use this software for any purpose,
+including commercial applications, and to alter it and redistribute it freely,
+subject to the following restrictions:
+
+1. The origin of this software must not be misrepresented; you must not claim that you wrote the original software. If you use this software in a product, an acknowledgment in the product documentation would be appreciated but is not required.
+2. Altered source versions must be plainly marked as such, and must not be misrepresented as being the original software.
+3. This notice may not be removed or altered from any source distribution.
+*/
+
+#include "btBulletDynamicsCommon.h"
+#include "BulletCollision/CollisionShapes/btHeightfieldTerrainShape.h"
+#include "BulletDynamics/MLCPSolvers/btDantzigSolver.h"
+#include "BulletDynamics/MLCPSolvers/btSolveProjectedGaussSeidel.h"
+#include "BulletDynamics/MLCPSolvers/btMLCPSolver.h"
+#include "BulletDynamics/Vehicle/btRaycastVehicle.h"
+#include "BulletDynamics/ConstraintSolver/btHingeConstraint.h"
+#include "BulletDynamics/ConstraintSolver/btSliderConstraint.h"
+#include "../CommonInterfaces/CommonExampleInterface.h"
+#include "LinearMath/btAlignedObjectArray.h"
+#include "btBulletCollisionCommon.h"
+#include "../CommonInterfaces/CommonGUIHelperInterface.h"
+#include "../CommonInterfaces/CommonRenderInterface.h"
+#include "../CommonInterfaces/CommonWindowInterface.h"
+#include "../CommonInterfaces/CommonGraphicsAppInterface.h"
+#include <stdio.h>
+#include "CarHandlingDemo.h"
+
+
+class CarHandlingDemo : public CommonExampleInterface
+{
+public:
+
+	CarHandlingDemo(struct GUIHelperInterface* helper);
+
+	void initPhysics();
+
+	void exitPhysics();
+
+	virtual ~CarHandlingDemo();
+
+	virtual void stepSimulation(float deltaTime);
+
+	virtual bool mouseMoveCallback(float x, float y)
+	{
+		return false;
+	}
+
+	virtual bool mouseButtonCallback(int button, int state, float x, float y)
+	{
+		return false;
+	}
+
+	virtual bool keyboardCallback(int key, int state);
+
+	virtual void renderScene();
+
+	virtual void physicsDebugDraw(int debugFlags);
+
+	virtual void resetCamera()
+	{
+		float dist = 5 * 8;
+		float pitch = -45;
+		float yaw = 32;
+		float targetPos[3] = { -0.33, -0.72, 4.5 };
+		guiHelper->resetCamera(dist, pitch, yaw, targetPos[0], targetPos[1], targetPos[2]);
```

Comment:
Seems like pitch and yaw arguments has changed placed - probably since this PR was issued.

guiHelper->resetCamera(dist, pitch, yaw, targetPos[0], targetPos[1], targetPos[2]);
=>
guiHelper->resetCamera(dist, yaw, pitch, targetPos[0], targetPos[1], targetPos[2]);